plugins/SitesManager/SitesManager.php

```diff
@@ -57,6 +59,7 @@ public function registerEvents()
             'SitesManager.deleteSite.end'            => 'onSiteDeleted',
             'System.addSystemSummaryItems'           => 'addSystemSummaryItems',
             'Request.dispatch'                       => 'redirectDashboardToWelcomePage',
+            'Template.noDataPageGTMTabInstructions' => 'noDataPageGTMTabInstructions'
```

Comment:
just a stylistic tweak
```suggestion
            'Template.noDataPageGTMTabInstructions'  => 'noDataPageGTMTabInstructions'
```